### ISSUE TITLE
Adding more information about taxes

### ISSUE THREAD

**Issue by CrafterSvK** [liberapay/liberapay.com#1949]
I think it would be nice if FAQ page included information about taxes, if there are exceptions for specific type of work, if there are limits of how much money can someone get from donations and if they should register in their country. 

There are number of reasons why leave it as is:
- responsibility, we don't need to take more responsibility as we already have
- language doesn't always equal country (maybe adding second page with specific country information (maybe only in specific languages?))
- out of date information
- translators are not always law students or experts in law (which they really don't need to be, but the info should be somewhat correct)

**Comment by revi** [comment]
Laws on taxes changes frequently (to some extent, year over year), and it is better to consult the real expert than believing what is written in a random website on the internet.

So no, I don't think we should include detailed "tax business" in our FAQ.

**Comment by CrafterSvK** [comment]
Laws really do change every year to some extent. However, I still think it is good to have some more general information as it is better than nothing.

We should:

- Include a date which it was updated per language (per country)
- Add a disclaimer that it might be wrong or misleading (and that they are on their own if they are doing something illegal).
- Point to right authorities and/or include links to specific government sites.

I don't think that it is that big of a deal. But it can at least point some things out.

**Comment by Avamander** [comment]
I'm not sure its worth the mantime to just replace a search engine. A disclaimer displaying "be aware of your local tax law" before allowing receiving donations would be apt however.

**Comment by trebmuh** [comment]
Agreed. It's [already in the FAQ](https://en.liberapay.com/about/faq#taxable).
